.buildkite/pipeline.yml

```diff
@@ -26,3 +26,18 @@ steps:
     command: |
       cp gradle.properties-example gradle.properties
       ./gradlew lintWordpressVanillaRelease
+  - label: "Test WordPress"
+    <<: *docker-container
+    command: |
+      cp gradle.properties-example gradle.properties
+      ./gradlew testWordpressVanillaRelease
```

Comment:
The call on `develop` is different https://github.com/wordpress-mobile/WordPress-Android/blob/a86331722f01844e69225ae46afc62d63d7f6f1f/.circleci/config.yml#L58

Why did you decide to drop the `--stacktrace` flag in all the steps and the `--no-daemon` here?

Reply:
re: `--stacktrace`; I find `stacktrace` to be useful in _very_ rare cases and it just adds noise otherwise. I am willing to bet that is the case for most people and would prefer to add it if it was requested/necessary.

re: `--no-daemon`; [Gradle officially recommends all builds to use the daemon](https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:disabling_the_daemon), and that includes CI. I am guessing this flag was passed to address an issue in the past, or it used to be recommended not to use the daemon in CI so maybe that advice was being followed, either way I don't think it's correct to use it right now. If daemon is failing, it's more likely that there is a problem on our end that should be fixed, but thankfully that's not the case right now.

Reply:
According to a quick git pickaxe, this `--no-daemon` was:
 - introduced by @hypest in https://github.com/wordpress-mobile/WordPress-Android/commit/c4c9f8d84f1e71d5702500b5f0232e7dc01cdca8
 - removed and introduced back by @JavonDavis in #13211 / https://github.com/wordpress-mobile/WordPress-Android/pull/13211/commits/399e78792379ca42e91291c231fe99e9863b9916 .

Maybe they can provide more context as to why this was added at that time? https://github.com/wordpress-mobile/WordPress-Android/pull/11833 seems to provide a lot of context too, which makes me tend to agree that this was something that might have been needed in the past as a workaround but can now safely be removed.

Reply:
Thanks for the extra context @AliSoftware @oguzkocer 🙌

Reply:
> Maybe they can provide more context as to why this was added at that time?

👋 friends, Olivier already shared the commits/PRs that are relevant and tried to capture the context there but, let me know if there's a specific question or unclear thing at this stage.